_posts/2022-01-20-code-reviews.md

```diff
@@ -28,7 +28,7 @@ With code reviews, at least two persons know something about the code and it red
 
 ## Better code quality
 
-> Code review remains the number one thing a company can do to improve code quality. - [State of Software Quality | Code Review 2021, Presented by SmartBear](https://smartbear.com/state-of-software-quality/code-review/)
```

Comment:
Here is how this appeared in blog:
![image](https://user-images.githubusercontent.com/73327769/150472526-3b7e8da3-bc9e-49c6-8603-beac3713c114.png)

Seems that pipe character isn't good idea inside the quotation.